airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/S3DestinationConstants.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.destination.s3;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+
+public final class S3DestinationConstants {
+
+  public static final int DEFAULT_PART_SIZE_MD = 5;
+  public static final DateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
suggest renaming this to `YYYY_MM_DD_FORMAT` for readability

Reply:
`YYYY` is week year. So I think we should keep using `yyyy`.

Personally I prefer `yyyy-MM-dd` because it is the ISO 8601 representation and I am used to it. But I am fine with changing the separator to underscore for readability.

Reply:
sorry I am just suggesting renaming the variable

Reply:
Oh, I just saw this comment. Will do.